base/simplebook/ansible/roles/simplebook/templates/supervisor-simplebook.conf.j2

```diff
@@ -0,0 +1,24 @@
+[program:simplebookserver]
+directory={{ simplebook_install_directory }}/services/api
+user:vagrant
```

Comment:
This user doesn't exist in production, so it should probably inherit from deployment_user?

Reply:
great catch ty